api/core/model_runtime/model_providers/__base/model_client_provider.py

```diff
@@ -0,0 +1,10 @@
+from typing import Any
+
+
+class ModelClientProvider:
+    @staticmethod
+    def get_service_client(credentials: dict = None, **kwargs: Any) -> Any:
```

Comment:
credentials parameter should be declared as optional

Reply:
OK, changed to `Optional[dict]`